sound/soc/intel/common/soc-acpi-intel-byt-match.c

```diff
@@ -197,8 +197,8 @@ struct snd_soc_acpi_mach  snd_soc_acpi_intel_baytrail_machines[] = {
 	{
 		.id = "10EC5682",
 		.drv_name = "sof_rt5682",
-		.sof_fw_filename = "intel/sof-byt.ri",
-		.sof_tplg_filename = "intel/sof-byt-rt5682.tplg",
+		.sof_fw_filename = "sof-byt.ri",
+		.sof_tplg_filename = "sof-byt-rt5682.tplg",
```

Comment:
@plbossart any particular reason you do this in a separate patch and not with the rest of the byt fixes? and similarly for the cnl rt5682 as well?

Reply:
yes. the first batch corresponds to what is already upstream and the second is still local to sof-dev. Splitting the two makes my life easier when pushing things upstream and merging back.